### ISSUE TITLE
I get a network error when inputing my own SQL into the text box

### ISSUE THREAD

**Issue by MattCremeens** [zhouxh19/LearnedRewrite#4]
![image](https://github.com/user-attachments/assets/ea97807e-e4c2-40e0-b84b-0fd2df8ce691)

**Comment by WilburDuncan** [comment]
Hi, I got it too, did u solve it?

**Comment by XuanheZhou** [comment]
sorry for the delayed response.

do you mean the online demo? We have closed the cloud server and may implement in another one.

**Comment by WilburDuncan** [comment]
Thanks for the reply Prof. Xuanhe, yes there's sth wrong with the online demo of http://rewrite_demo.dbmind.cn/, I notice that now we can use the REWRITE but ChatGPT or ChatBot is not working, for the "another one" you said could you share the link (or is it still under development)?

**Comment by XuanheZhou** [comment]
yes, we have restarted the service (http://rewrite_demo.dbmind.cn). And we no long provide the free llm-based service (you can check out our demo video and implement it with local llms).
